bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxServerHandler.java

```diff
@@ -998,6 +1019,62 @@ private void updateChannelFavoritesList(List<Favorite> favorites) {
                 updateState(CHANNEL_FAVORITES_LIST, new StringType(favoritesList));
             }
         }
+
+        private void scheduleRequestFavorites() {
+            // Delay the execution to give the player thing handlers a chance to initialize
+            scheduler.schedule(SqueezeBoxServerHandler.this::requestFavorites, 3L, TimeUnit.SECONDS);
```

Comment:
You should cancel the future this produces in case this handler gets disposed.

Reply:
Ok, done. The `isDone` may not be necessary, but I added it for completeness.